menu.py

```diff
@@ -357,7 +331,7 @@ def draw(self):
                     pyxel.rect(21, 11, 158, 182, pyxel.COLOR_BLACK)
                     pyxel.text(85, 15, "Factions", pyxel.COLOR_WHITE)
                     pyxel.text(25, 30, str(self.faction_colours[self.faction_wiki_idx][0].value),
-                               self.faction_colours[self.faction_wiki_idx][1])
+                            self.faction_colours[self.faction_wiki_idx][1])
```

Comment:
While you're fixing the lint, this indentation change should be reverted.

Reply:
[841bd66](https://github.com/ChrisNeedham24/microcosm/pull/91/commits/841bd66dbd0cc1c655d54ad409b59603d697ddd2)

Reply:
[c8f4e20](https://github.com/ChrisNeedham24/microcosm/pull/91/commits/c8f4e200da76f53b0aea67e242812e7d0ff8f24d)